include-fragment-element.js

```diff
@@ -6,7 +6,7 @@
   function fire(name, target) {
     setTimeout(function() {
       var event = target.ownerDocument.createEvent('Event');
-      event.initEvent(name, true, true);
+      event.initEvent(name, false, false);
```

Comment:
Since this is a significant API change, I think maybe a major version bump is in order.